@navikt/core/react/src/form/combobox/Input/Input.tsx

```diff
@@ -184,10 +215,11 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
       <input
         {...rest}
         {...omit(inputProps, ["aria-invalid"])}
-        ref={ref}
+        ref={mergedRefs}
         value={value}
         onBlur={() => virtualFocus.moveFocusToTop()}
-        onChange={onChangeHandler}
+        onClick={() => value && updateInputValue(value)}
```

Comment:
`externalOnChange()` is not being called if you "complete" the selection by clicking. Is that intended?
(I.e. should we call `onChange` instead of `updateInputValue` here?)

Reply:
Enig. Går for følgende:
```suggestion
        onClick={() => value && onChange(value)}
```